static/src/javascripts/bootstraps/commercial.js

```diff
@@ -89,7 +89,8 @@ define([
             ['cm-hostedOJCarousel', hostedOJCarousel.init]);
     }
 
-    if (!config.page.disableStickyTopBanner) {
+    var supportsSticky = document.documentElement.classList.contains('has-sticky');
+    if (!config.page.disableStickyTopBanner && !supportsSticky) {
```

Comment:
Do you think its a little confusing now with have several sticky flags per page? I count three, `top-banner-ad-container--not-sticky` for the container, `!has-sticky` on the document, and `disableStickyTopBanner` in JS config.